common/src/Model/BrushFace.h

```diff
@@ -154,6 +154,8 @@ namespace TrenchBroom {
             vm::vec3 center() const;
             vm::vec3 boundsCenter() const;
             FloatType area(vm::axis::type axis) const;
+            FloatType area() const;
```

Comment:
I'd suggest we rename the existing function to "axialArea" or "projectedArea" to differentiate it in a separate commit.